kikar_hamedina/core/urls.py

```diff
@@ -17,6 +17,7 @@
 from core.models import MEMBER_MODEL, PARTY_MODEL
 from core.rss_feeds import LatestStatusesRSSFeed, PartyRSSFeed, MemberRSSFeed, KeywordsByUserRSSFeed, CustomQueryRSSFeed
 from django.contrib.sitemaps.views import sitemap
+#from . import video_type
```

Comment:
This is a mistake. it is not relevant. pls do not update the main repository with this file.